cypress/integration/tests/developer/applicationinventory/assessment/assess_review.test.ts

```diff
@@ -125,6 +126,32 @@ describe("Application assessment and review tests", { tags: "@tier1" }, () => {
         cy.wait(2000);
         application.verifyStatus("review", "Completed");
 
+        // Delete application
+        application.delete();
+        cy.wait(2000);
+    });
+    it("Enable and Disable Review without assessment", function () {
+        // Enable allow review without running assessment
+        let generalConfig = GeneralConfig.getInstance();
```

Comment:
**nitpick:** This is not an error but good practice, if you are reassigning the value of a variable, it can be declared as `const` to improve readability.

https://stackoverflow.com/a/41086682/7491884

Reply:
Thanks for the suggestion @abrugaro  const being added